tests/testthat/test-modify-list.R

```diff
@@ -0,0 +1,25 @@
+context("modify_list")
```

Comment:
Test files should match R files; so you should either rename this or pull `modify_list()` into its own file.